.github/workflows/manual-backport.yml

```diff
@@ -0,0 +1,85 @@
+name: manual-backport
+on:
+  workflow_dispatch:
+      inputs:
+        target_branch:
+          description: 'Branch to backport to (e.g. 8.0)'
+          required: true
+        commit_sha:
+          description: 'Sha256 hash of the merge commit to use in backporting'
+          required: true
+        exceptions:
+          description: 'Comma seperated list of files to skip staging e.g. detection_rules/etc/packages.yml,detection_rules/attack.py)'
+          required: false
+
+jobs:
+  commit:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Validate the source branch
+        uses: actions/github-script@v3
+        with:
+          script: |
+            if ('refs/heads/main' !== '${{github.event.ref}}') {
```

Comment:
Do we really want to limit this to only backporting from main?

Reply:
Great question. We shouldn't do this anymore. I removed this main check.